### PR TITLE
Accessibility bug fixes

### PR DESCRIPTION
**Pull request by sbanni**
Fixes for 395754, 395756, 395808, and 395744

### CODE REVIEW

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs [hunk 1]
```diff
@@ -23,6 +24,25 @@
 
 namespace NuGet.PackageManagement.UI
 {
+    public class InfiniteScrollListListBox : ListBox
```
Comment (alpaix) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs]: One class per file. Also can this class be internal?

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs [hunk 2]
```diff
@@ -23,6 +24,25 @@
...
+    {
+        protected override AutomationPeer OnCreateAutomationPeer()
+        {
+            return new InfiniteScrollListListBoxAutomationPeer(this);
+        }
+    }
+
+    public class InfiniteScrollListListBoxAutomationPeer : ListBoxAutomationPeer
```
Comment (alpaix) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs]: the same.

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs (hunk 1 above)
Comment (alpaix) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs]: Simplify to `InifiniteScrollListBox`?

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Options/GeneralOptionControl.Designer.cs [hunk 3]
```diff
@@ -36,14 +36,12 @@ private void InitializeComponent()
             this.BindingRedirectsHeader = new System.Windows.Forms.Label();
             this.localsCommandButton = new System.Windows.Forms.Button();
             this.localsCommandStatusText = new System.Windows.Forms.RichTextBox();
-#if !VS14
```
Comment (alpaix) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Options/GeneralOptionControl.Designer.cs]: `#if` should stay as these options are not applicable to VS2015

**Review by alpaix — CHANGES_REQUESTED** [review]

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs (hunk 1 above)
Comment (sbanni) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs]: Done.

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs (hunk 2 above)
Comment (sbanni) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs]: Done

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Options/GeneralOptionControl.Designer.cs (hunk 3 above)
Comment (sbanni) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Options/GeneralOptionControl.Designer.cs]: The designer automatically removed those, will go back and make sure unnecessary changes are not done like that

**Review by alpaix — APPROVED** [review]
LGTM

**Pull request merged**